cmd/reportsStorage/main.go

```diff
@@ -0,0 +1,193 @@
+package main
+
+//go:generate sh -c "mkdir -p ../../gen/reportsStorage && swagger generate server -A reportsStorage -t ../../gen/reportsStorage -f ../../docs/api/reportsStorage.yml --exclude-main --principal string"
+
+import (
+	"context"
+	"encoding/base64"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"log"
+	"os"
+	"os/signal"
+	"syscall"
+
+	loads "github.com/go-openapi/loads"
+	flags "github.com/jessevdk/go-flags"
+	"github.com/rs/cors"
+	"github.com/rs/zerolog"
+
+	"github.com/iryonetwork/wwm/gen/reportsStorage/restapi"
+	"github.com/iryonetwork/wwm/gen/reportsStorage/restapi/operations"
```

Comment:
could not import github.com/iryonetwork/wwm/gen/reportsStorage/restapi/operations (cannot find package "github.com/iryonetwork/wwm/gen/reportsStorage/restapi/operations" in any of:
	/tmp/tmp.jNUNwZeI6Z/src/github.com/iryonetwork/wwm/vendor/github.com/iryonetwork/wwm/gen/reportsStorage/restapi/operations (vendor tree)
	/usr/local/go/src/github.com/iryonetwork/wwm/gen/reportsStorage/restapi/operations (from $GOROOT)
	/tmp/tmp.jNUNwZeI6Z/src/github.com/iryonetwork/wwm/gen/reportsStorage/restapi/operations (from $GOPATH))